openshift/TestConnections.txt

```diff
@@ -0,0 +1,6 @@
+# Uses the 'testConnection' script from https://github.com/BCDevOps/openshift-developer-tools
+# Example:
+# echo -e "backup-postgres"  | xargs -I {} testConnection -n a99fd4-dev -f TestConnections.txt -p {}
+
+google.com:80
+issuer-kit-wallet:5432
```

Comment:
This file should not be in the repository as far as I am concerned, it is deployment specific.